integration/helpers/create-fixture.ts

```diff
@@ -327,11 +327,6 @@ async function writeTestFiles(init: FixtureInit, dir: string) {
       let filePath = path.join(dir, filename);
       await fse.ensureDir(path.dirname(filePath));
       let file = init.files![filename];
-      // if we have a jsconfig we don't want the tsconfig to exist
-      if (filename.endsWith("jsconfig.json")) {
-        let parsed = path.parse(filePath);
-        await fse.remove(path.join(parsed.dir, "tsconfig.json"));
-      }
```

Comment:
Couldn't someone still technically add one via the fixture setup?  I don't have a specific use-case offhand but it it worth removing this if it prevents that in the future?

```js
fixture = await createFixture({
  files: {
    "jsconfig.json": ...
  }
}
```